### DISCUSSION TITLE
電機系介紹文(正經版)

### DISCUSSION THREAD

**Discussion by HsuSungTing** [1101-GS4719-C/Python-Community-GS4719-C#1200] [General]
既然介紹土木系，那身為一個迷茫的電機系學生，來整理一下電機系到底在幹嘛吧:

電機系是一個廣泛的學科，涵蓋了許多不同的領域，例如電子學、控制、通訊、功率、電磁學等等。然而，對於大眾來說，可能存在著一些關於電機系的誤解。(不要再叫我修水電了，我沒有比較會QQ)

首先，一些人可能會誤以為電機系只是學習如何修理電器或者是使用一些常見的電子產品。實際上，電機系是一門非常深奧的學科，需要學習的知識非常廣泛，包括物理學、數學、計算機科學、電子學等等。除了學習理論知識外，還需要掌握各種軟件和硬件工具，以便進行設計和實驗。(大家可以進來體會接麵包板和一堆Mos有多快樂 : ) )

另外，一些人可能誤以為念電機系就能很快找到工作。實際上，電機系是一個競爭激烈的領域，需要具備扎實的理論知識、實踐能力和創新思維。在現代科技迅速發展的背景下，電機系的知識也在不斷地更新和演進，需要不斷地學習和進修。(白話文:隨各種工具演變，我學到的東西可能幾年內就要全部重學ㄌ....)

在台灣，電機系的課程包括電子學、控制系統、通訊系統、電力系統等等，並且還可以選擇特定領域的課程來深入學習。台灣的電機系有著深厚的學術背景和豐富的研究資源，可以讓學生在這個領域中獲得全面的學習和研究經驗。

未來出路方面，隨著科技的不斷發展和創新，電機系的就業前景也越來越廣泛。從電子產品設計、半導體技術、網絡通信、智能裝置、機器人技術、能源綠色化等，都有著相當的就業需求。在科技領域中，電機系的學生們可以發揮他們的專業知識和技能，參與到科技創新的研究和開發項目中，尤其是在人工智慧、大數據和物聯網等新興技術領域。(題外話， chatgpt除了很寫code，也很會解電子學，我要躺平ㄌ)

**Comment by DuxtinTai** [comment]
我也是一個電機系的學生，看到你整理的這篇關於電機系的文章感到非常認同。的確，許多人對於電機系存在著一些誤解，以為這個學科只是在學習如何修理電器或者是使用一些常見的電子產品。但實際上，電機系是一門非常深奧的學科，需要學習的知識非常廣泛，同時各個領域也在不斷地更新和演進，需要不斷地學習和進修。
剛剛去實驗了chatgpt解電子學，還真的可以...
不過我相信培養扎實的基本功依然非常重要，畢竟對於某些需要具體實踐經驗的問題，chatgpt仍然無法提供具體有效的幫助，反過來想，若我們可以掌握好，這同時也是我們的優勢所在。

**Comment by wanderer657** [comment]
1.電機系的學習涉及多個領域，包括電子學、控制系統、通訊技術等。
2.電機系的學習培養了學生的綜合思維能力和問題解決能力。
3.隨著科技的發展，電機工程領域將不斷涌現新的挑戰和機遇。
4.電機工程師需要具備扎實的專業知識和解決問題的能力。

**Comment by SusuTima** [comment]
電機的領域本身很廣，自控、通訊、ic設計、生醫、電力系統，每個領域要學的東西都很多。而台積電之所以可以成為護國神山，就是強在可以將晶片越做越小（目前3奈米？），所以電機系的學生要有很深厚的基礎知識，才有辦法做後續的研究和工作。而臺灣在晶片產業中主要負責晶片製成（台積電）和封裝（日月光）和少部分的晶片設計，美國則是主掌了主要上游的晶片設計。以上如果有任何錯誤請糾正，感謝。

**Comment by EPHMR0524** [comment]
個人身為電機系的學生，我覺得版主的介紹是簡單卻非常精確，我並非從大一就是電機系的學生，我是二年級才轉入的，若要和我大一時所在的科系做比較的話，我可以明確感受到的是電機系的課能讓我感受到相當"疲倦"的氣息，彷彿每個人昨晚都熬夜一樣(好啦我也是)，每個人都是非常努力，只要稍有怠惰可能排名就不知道被卷去哪裡，老實說是蠻累的，我覺得成績如果不好的話沒有愛真的讀不下去
如果是現在的我，應該是完全不想推薦新生讀電機系，真的累死，但出去工作拿到錢之後我可能會改觀吧。

**Comment by KH1126** [comment]
我在修習電機系的選修科目，如資料結構、VLSI、演算法等時，我深刻意識到要想在這個領域有所發展，所需要的知識儲備量需要相當大。這些選修科目不僅僅是知識的積累，更是開啟了我對電機領域更深層次探索的大門。特別是在 VLSI 領域，不僅需要對積體電路的原理和設計有深入的理解，還需要掌握相關的軟件工具和技術，以應用於實際的設計和製造過程中。

因此我認為即是想要在電機領域發展，在平時的學習中，需要持續不斷地學習和探索，才能在未來的職業生涯中取得更好的成就。

**Comment by benbebobo149** [comment]
我也覺得電機系會比資工系的出路廣很多，我看到很多電機系最後跳槽做純軟，但很少看到資工系跳過去做硬體，可能是因為我們課程本來很少排除一些物理相關的課程。這讓我有些擔憂，因為資工系的同學可能會面臨著更有限的職業選擇。感謝同學的分享，讓我思考到了這個問題。

**Comment by LinChloe** [comment]
我曾經看過電機系的卷哥，看完他上課的態度後我只能甘拜下風。
真的不得不承認讀電機系的都是狠人。
期待未來你們對台灣未來的貢獻。
(以前上課我對連接麵包板蠻有興趣的，但你說完感覺不太妙)

**Comment by thomasxshen** [comment]
電機工程學是研究電力、電子和電磁學的應用科學。它涵蓋從微小的微處理器到巨大的發電機組的廣泛技術，對現代社會至關重要。電機工程師設計、開發、測試和監督電子設備和系統的製造，包括通訊系統、電腦和大型電力網絡。這個領域不斷創新，推動技術進步。

**Comment by willychiuchiu** [comment]
電機系未來出路很廣，不管是成為軟體工程師或者硬體工程師都大有人在。目前讀的這兩年下來，深深感受到每堂課的紮實訓練，真的是必須用肝來讀的科系，畢竟電機系就是在訓練工程師，累一點真的是不可避免的⋯

**Comment by yyujimmy** [comment]
電機系的確是一個充滿挑戰和機遇的領域！從事電機工程需要具備廣泛的知識和技能，但隨之而來的也是豐富多彩的職業發展前景。從電路設計到通訊系統，從控制系統到智能裝置，每個領域都有著無限的可能性和發展空間。

你所提到的學習過程確實需要不斷地學習和更新知識，但這也是電機工程師的樂趣所在。隨著科技的發展，每一次的學習和挑戰都是為了更好地應對未來的需求和挑戰。而對於擁有扎實知識和技能的電機工程師來說，未來的發展前景是非常值得期待的。

在這個快速發展的時代，我們對於科技的需求也越來越多元化，而電機工程師正是擁有解決這些挑戰所需的關鍵人才。因此，我相信電機系的學生們將會在未來的職業生涯中取得令人矚目的成就！
